docs/performance/benchmark.md

```diff
@@ -0,0 +1,30 @@
+# Benchmark
+
+This document will show and explain the benchmark of an approximate nearest neighbor on the Vald cluster, such as [ann-benchmarks](https://github.com/erikbern/ann-benchmarks), and also explain how to perform the ann-benchmark test on your Vald cluster.
+
+If you want to perform load testing on the Vald cluster, please refer to [this document](loadtest.md).
+
+<!--
+We also created a guideline of the unit benchmark testing, please refer to [this document](unit_benchmark.md).
+-->
+
+## Overview
+
+In this document, we will perform the [ann-benchmarks](https://github.com/erikbern/ann-benchmarks) on Vald cluster.
+
+The test will perform on the whole Vald cluster, which means the request will broadcast to multiple Vald agents instead of single Vald agent, and return the aggreated result.
```

Comment:
```suggestion
The test will perform on the whole Vald cluster, which means the request will broadcast to multiple Vald agents instead of a single Vald agent, and return the aggregated result.
```